src/main/java/db/Database.java

```diff
@@ -0,0 +1,9 @@
+package db;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+
+public class Database {
+    public static final List<FruitTransaction> database = new ArrayList<>();
```

Comment:
Our db should save some data about fruits not about transactions. Lets implement some Map<String, Integer> that will store fruit name and fruit quantity